workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/services/tickets.ts

```diff
@@ -123,3 +125,48 @@ export async function generateNewTicket() {
 		method: 'POST',
 	});
 }
+
+export async function updateTicketStatus(ticket: Ticket) {
+	let listTypeDefinitions = {} as ListTypeDefinitions;
+
+	if (!(J3Y7_PRIORITIES in listTypeDefinitions)) {
```

Comment:
This will always evaluate to true, since you are declaring `listTypeDefinitions` in the function.